addon/package.json

```diff
@@ -0,0 +1,77 @@
+{
+  "name": "ember-animated-tools",
+  "version": "0.5.0",
+  "description": "Components that help you develop and debug ember-animated transitions",
+  "keywords": [
+    "ember-addon"
+  ],
+  "repository": "https://github.com/ember-animation/ember-animated-tools",
+  "license": "MIT",
+  "author": "Edward Faulkner <edward@eaf4.com>",
+  "exports": {
+    ".": "./dist/index.js",
+    "./*": "./dist/*",
+    "./addon-main.js": "./addon-main.js"
+  },
+  "directories": {
+    "doc": "doc",
+    "test": "tests"
+  },
+  "files": [
+    "addon-main.js",
+    "dist"
+  ],
+  "scripts": {
+    "build": "rollup --config",
+    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel \"lint:!(fix)\"",
+    "lint:fix": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*:fix",
+    "lint:hbs": "ember-template-lint .",
+    "lint:hbs:fix": "ember-template-lint . --fix",
+    "lint:js": "eslint . --cache",
+    "lint:js:fix": "eslint . --fix",
+    "prepublishOnly": "rollup --config",
+    "start": "rollup --config --watch",
+    "test": "echo 'Please refer to test-app for addon tests'"
+  },
+  "dependencies": {
+    "@embroider/addon-shim": "^1.5.0",
+    "ember-animated": "file:../ember-animated-0.12.0.tgz"
```

Comment:
As `ember-animated` a monorepo, we can't use `github:ember-animation/ember-animated#v2-addon` scheme.

as temp workaround, packed `ember-animated` and used here.

Once `ember-animated` published to npm (via beta or actual release), will have a follow up PR to point back to npm.